src/forms/BoundCheckboxField.test.tsx

```diff
@@ -63,6 +63,33 @@ describe("BoundCheckboxField", () => {
     click(r.isAvailable);
     // Then the callback should be triggered with the current value
     expect(autoSave).toBeCalledWith(false);
+    expect(autoSave).toBeCalledTimes(1);
```

Comment:
Was this a merge conflict or something? Changes seem unrelated?

Reply:
Ah yep sorry, this slipped in from some tests I was doing for a different ticket